lib/src/generated/client.dart

```diff
@@ -427,7 +427,7 @@ class MistralaiClientDartClient {
       requestType: '',
       responseType: 'application/json',
     );
-    return ModelCard.fromJson(_jsonDecode(r) as Map<String, dynamic>);
+    return UnionModelCard.fromJson(_jsonDecode(r) as Map<String, dynamic>);
```

Comment:
I think we can also make it so the name won't change. I'll reach out to you on chat.

Reply:
In [2b39735](https://github.com/nomtek/mistralai_client_dart/pull/102/commits/2b39735fc3b0a199bbce1cdce849a17e419884b4), I updated code generation script as discussed on slack.